docker/fusionauth/getit

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+# Copyright (c) 2018-2023, FusionAuth, All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+#
+# You can install FusionAuth by running:
+#
+# curl -s https://fusionauth.io/getit | sh -
+#
+
+# Useful colors
+COLOR_RESET="\e[0m"
+COLOR_RED="\e[31m"
+COLOR_GREEN="\e[32m"
+COLOR_YELLOW="\e[33m"
+COLOR_CYAN="\e[36m"
+
+# Displays help menu
+display_help() {
+  printf "%bDescription:%b\n" "$COLOR_YELLOW" "$COLOR_RESET"
+  printf "  Downloads necessary files to start a FusionAuth instance\n\n"
```

Comment:
we should be pretty explicit here about what files we are downloading, and I would make the default behavior "help" instead of execute. Slightly less slick but better for telling people what we are doing up front